barretenberg/cpp/src/barretenberg/honk/composer/ultra_composer.hpp

```diff
@@ -62,24 +54,23 @@ template <UltraFlavor Flavor> class UltraComposer_ {
     UltraComposer_& operator=(UltraComposer_ const& other) noexcept = default;
```

Comment:
You've noticed in the files i've added that explicit constructors are not needed. I only added them because they were also present here but it seems that they are not needed here either (by seems i mean code compiles and tests pass if I remove them). Can they be removed?